epoxy/settings.py

```diff
@@ -117,3 +105,24 @@ def encode(self, value):
 
     def decode(self, value):
         return value
+
+
+class EnvironmentSetting(BaseSetting):
```

Comment:
Although the others lack it presently, I think we should probably start adding more full docstrings for public items like this that explain usage.  For `EnvironmentSettting`, I think describing the precedence ordering of the following is probably the key piece of information a user would need to understand how this settings works:

* Default specified in code
* Setting value specified in YML configuration
* Environment variable value